### ISSUE TITLE
Update bower/component.json

### ISSUE THREAD

**Issue by brianmhunt** [SteveSanderson/knockout.mapping#127]
It seems the [Bower](http://twitter.github.com/bower/) component.json/package is quite outdated, using `knockout.mapping` version 2.0.1. :ghost:

Whenever convenient, a bump would be wonderful. :)

**Comment by sagacity** [comment]
I am not the author of the Bower package though, so... what actions need to be taken?

**Comment by brianmhunt** [comment]
Thanks @RoyJacobs . Oddly there are two knockout mapping plugins `knockout-mapping` and `knockout.mapping`.

knockout.mapping links to //github.com/RayKwon/knockout.mapping.git
knockout-mapping links to //github.com/SteveSanderson/knockout.mapping.git

The only action typically required is to add a tag for the latest version, then Bower will do all the work.

It seems as though since I filed the bug a tag for `knockout-mapping` 2.3.5 has been added, and so that version is now available on Bower. Awesome - closing this issue. Thanks to whoever did that. :)

Cheers
